google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -366,7 +366,9 @@ TEST(MakePublisherConnectionTest, TracingEnabled) {
 
   EXPECT_THAT(
       span_catcher->GetSpans(),
-      Contains(SpanNamed("projects/test-project/topics/test-topic send")));
+      ::testing::UnorderedElementsAre(
```

Comment:
style nit: we try to keep `::testing` out of the expectations, so that they read slightly more like English

So `using ::testing::UnorderedElementsAre;` somewhere

Reply:
Done